teitelbot/response/classifier.py

```diff
@@ -0,0 +1,60 @@
+import re
+from random import random
+from typing import List
+
+import config
+from constants.significant_words import words_to_repeat_sarcastically
+from response.action import Action
+from util import string_utils
+
+
+def classify(text: str) -> Action:
+    return possible_classifications(text)[0]
+
+
+def possible_classifications(text: str) -> List[Action]:
+    classifier_to_action = {
+        _check_number: Action.CHECK_NUMBER_ACTION,
```

Comment:
Indent to make pretty

Reply:
Or is that impossible because python sucks?

Reply:
![image](https://user-images.githubusercontent.com/24360792/104855448-82e28280-5915-11eb-9c1f-4435f2a67301.png)